frida_tools/repl.py

```diff
@@ -67,7 +66,7 @@ def _add_options(self, parser):
             parser.add_option("-o", "--output", help="output to log file", dest="logfile", default=None)
 
         def _initialize(self, parser, options, args):
-            self._user_script = options.user_script
+            self._user_script = os.path.abspath(options.user_script)
```

Comment:
I guess this will crash if `options.user_script` is `None` because `-l` wasn't specified?